css/modals/video-quality/_video-quality.scss

```diff
@@ -162,11 +162,11 @@
 }
 
 .centeredVideoLabel.moveToCorner {
-    z-index: $tooltipsZ;
+    z-index: $zindex3;
 }
 
 #videoResolutionLabel {
-    z-index: #{$tooltipsZ + 1};
+    z-index: $zindex3 + 1;
```

Comment:
These values had been bumped up when vertical filmstrip 1:1 mode needed to wrestle with the video quality label but that doesn't happen anymore as the video quality label never has to overlap filmstrip anymore. These styling changes followed removal of some unnecessary CSS making the filmstrip higher in z-index. Z-indexing in general needed to be lowered for filmstrip due to the new toolbar having popup menus that need to overlap the filmstrip.